arch/csr/tdata1.yaml

```diff
@@ -0,0 +1,63 @@
+# yaml-language-server: $schema=../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: tdata1
+long_name: Trigger Data 1
+address: 0x7A1
+priv_mode: M
+length: XLEN
+description: |
+  This register provides access to the trigger selected by `tselect`.
+  The reset values listed here apply to every underlying trigger.
+
+  This register is optional if no triggers are implemented.
+  Writing 0 to this register must result in a trigger that is disabled.
+  If this trigger supports multiple types, then the hardware should disable it
+  by changing type to 15.
+
+definedBy: Sdtrig
+fields:
+  TYPE:
+    location_rv32: 31-28
+    location_rv64: 63-60
+    type: RW
+    description: |
+      0 (none):: There is no trigger at this `tselect`.
+      1 (legacy):: The trigger is a legacy SiFive address match trigger. These should not be implemented and aren’t further documented here.
+      2 (mcontrol):: The trigger is an address/data match trigger. The remaining bits in this register act as described in `mcontrol`.
+      3 (icount):: The trigger is an instruction count trigger. The remaining bits in this register act as described in `icount`.
+      4 (itrigger):: The trigger is an interrupt trigger. The remaining bits in this register act as described in `itrigger`.
+      5 (etrigger):: The trigger is an exception trigger. The remaining bits in this register act as described in `etrigger`.
+      6 (mcontrol6):: The trigger is an address/data match trigger. The remaining bits in this register act as described in `mcontrol6`. This is similar to a type 2 trigger, but provides additional functionality and should be used instead of type 2 in newer implementations.
+      7 (tmexttrigger):: The trigger is a trigger source external to the TM. The remaining bits in this register act as described in `tmexttrigger`.
+      12—14 (custom):: These trigger types are available for non-standard use.
+      15 (disabled):: This trigger is disabled. In this state, `tdata2` and `tdata3` can be written with any value that is supported for any of the types this trigger implements. The remaining bits in this register, except for `DMODE`, are ignored.
+
+      Other values are reserved for future use.
+    reset_value: UNDEFINED_LEGAL
+  DMODE:
+    location_rv32: 27
+    location_rv64: 59
+    type: RW
+    description: |
+      If type is 0, then this bit is hard-wired to 0.
```

Comment:
I think you'll need a blank line here, too. I don't know if there is currently any (working) rendering that'll show how this looks?